RealmTasks Android/app/src/main/java/io/realm/realmtasks/TaskActivity.java

```diff
@@ -184,21 +181,16 @@ public boolean onClicked(ItemViewHolder viewHolder) {
         @Override
         public void onChanged(ItemViewHolder viewHolder) {
             adapter.onItemChanged(viewHolder);
-            adapter.notifyItemChanged(viewHolder.getAdapterPosition());
         }
 
         @Override
         public void onAdded() {
             adapter.onItemAdded();
-            adapter.notifyItemInserted(0);
```

Comment:
If the row is inserted, colors of other rows should be changed. So `notifyDataSetChanged()` should also be added for pull down to add the row.

Because there is a visual artifact during pulling down, I am investigating this issue.